Mention request/response bodies for GraphQL should be guarded by a flag (#1003)
getsentry/develop: src/docs/sdk/features.mdx

}
```

> The `data` field is a JSON object that contains the GraphQL response payload. Attaching request and response bodies should be guarded by `sendDefaultPii` or another flag  to opt-in (whichever makes more sense for the specific SDK).

Reviewer: ```suggestion
The `data` field is a JSON object that contains the GraphQL response payload. Attaching request and response bodies should be guarded by `sendDefaultPii` and/or another flag to opt-in (e.g. `captureFailedRequests`).
```

Ability for the SDK to attach request body to events and triggered during the execution of request.

> Attaching request and response bodies should be guarded by `sendDefaultPii` or another flag  to opt-in (whichever makes more sense for the specific SDK).

Reviewer: @adinauer this is under `Attaching Request Body in Server SDKs` which is already opt-in because `maxRequestBodySize` defaults to `none`.
If this is about the GraphQL integration, let's add this specifically to that section.
Otherwise, adapt this section to Request/Response, and mention that this applies to GraphQL as well.

Author: Moved it up
